src/Events/Services/LabTransmitEvent.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ *   package   OpenEMR
+ *   link      http://www.open-emr.org
+ *  author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  Copyright (c)
+ *  All rights reserved
+ *
+ */
+
+namespace OpenEMR\Events\Services;
+
+use Symfony\Contracts\EventDispatcher\Event;
+
+class LabTransmitEvent extends Event
+{
+    /**
+     * This event is triggered when a lab is to be transmitted
+     */
+    const EVENT_LAB_TRANSMIT = 'lab.transmit';
+
+    /**
+     * This event is triggered when a lab requisition form is returned from Quest
+     * Requisition form has to be enabled in the globals
```

Comment:
If we think this event will only apply to Quest, we should rename this class to be QuestLabTransmitEvent.

Reply:
If its a generic event, probably need to change up the comment and just mention quest as being an example of where this is triggered.

Reply:
Quest provides a requisition form via API call whereas Labcorp used to have to be built by EHR. We'd want this event to be generic and able to be triggered for a Labcorp eReq(or any other) so please change now so we won't have to move it later.
How are you handling user request for results. Is there an event already in place for electronic report?

Reply:
When I was building this feature, all of you are playing like a record in my head to make this flexible and not to hardcode things into place. Use the dispatch system to couple the system loosely. That is what I was trying to do. I discovered that Labcorp requisition is built into the system. It is buried in there. I found it during my build process. So, I used the dispatch to send the lab order over to the module to send out of the API connection to Quest. 
All the module does is handle the communication between OpenEMR and Quest. All other processes are the same. I did not add anything to the codebase. I did not have to. I knew from all my years of working with the codebase what it was capable of doing. I did not reinvent the wheel. I was standing on the shoulders of Jerry and Rob. The only thing I needed to do was to get the order out to Quest and retrieve results via an API call.

Reply:
And I can appreciate your efforts to follow those guidelines Sherwin, I do! But this is a singular event and you missed my whole naming convention concern. This event should have been put on the button click event and qualified in the handler by passing the lab id! Then you are following the flavor of useful events that the maintainers have to be mindful of

Reply:
@adunsulag took charge of this review so I leave it to him. Be mindful though Sherwin there is a good possibility this event will get modified in the future!

Reply:
Looks like the button click event and generic lab id thing could be handled in another PR.  With the  new class name I believe this resolves the issue.